async-nats/src/lib.rs

```diff
@@ -324,6 +324,15 @@ pub struct ServerInfo {
     /// Whether server goes into lame duck mode.
     #[serde(default, rename = "ldm")]
     pub lame_duck_mode: bool,
+    /// Name of the cluster if the server is in cluster-mode
+    #[serde(default, skip_serializing_if = "Option::is_none")]
```

Comment:
No need for `serialize` attributes, as we only `deserialize` `ServerInfo`

Reply:
Fixed, thanks, indeed, only `Deserialize` is derived, didn't catch that :)